### PR TITLE
VTCCA-3043: make sure agentAppointed field is send to the backend

### PR DESCRIPTION
**Pull request by habeeb01**

### CODE REVIEW

File: app/controllers/agentAppointment/AppointAgentController.scala [hunk 1]
```diff
@@ -100,13 +100,17 @@ class AppointAgentController @Inject()(
...
-        request.organisationAccount.id, agentOrganisation.fold(throw new IllegalArgumentException("agent organisation required."))(_.id))
+      response <- agentRelationshipService.getMyOrganisationPropertyLinksWithAgentFiltering(
+        params = params,
+        pagination = AgentPropertiesParameters(
+          agentCode = agentCode,
+          checkPermission = AgentPermission.fromName(checkPermission).getOrElse(StartAndContinue),
+          challengePermission = AgentPermission.fromName(challengePermission).getOrElse(StartAndContinue),
+          pageNumber = pagination.page,
+          pageSize = pagination.pageSize,
+          agentAppointed = agentAppointed.getOrElse("BOTH")
```
Comment (darrell-plant) [inline app/controllers/agentAppointment/AppointAgentController.scala]: what is "BOTH"?

Comment (habeeb01) [reply app/controllers/agentAppointment/AppointAgentController.scala]: we default to BOTH because that will bring back all of the user's properties

Comment (habeeb01) [reply app/controllers/agentAppointment/AppointAgentController.scala]: BOTH is the default on the API as well

Comment (darrell-plant) [reply app/controllers/agentAppointment/AppointAgentController.scala]: then `AgentPropertiesFilter.Both.name` ?

Comment (habeeb01) [reply app/controllers/agentAppointment/AppointAgentController.scala]: updated

**Comment by darrell-plant** [comment]
LGTM 👍

**Comment by jon-pike** [comment]
👍

**Pull request merged**